src/blocktypes/arion.nix

```diff
@@ -19,8 +19,9 @@
       flake,
       fragment,
       fragmentRelPath,
+      target,
     }: let
-      cmd = "arion --prebuilt-file $(nix build ${flake}#${fragment}.config.out.dockerComposeYaml --print-out-paths)";
+      cmd = "arion --prebuilt-file ${target.config.out.dockerComposeYaml}";
```

Comment:
Is the removal of  ` --print-out-paths` motivated by mintenance / removal of techincal debts?

Reply:
We need the outpath, iterpolating the derivation gives us the outpath, no printing required